src/ansys/mapdl/core/cli/hpc.py

```diff
@@ -0,0 +1,271 @@
+# Copyright (C) 2024 ANSYS, Inc. and/or its affiliates.
+# SPDX-License-Identifier: MIT
+#
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+
+"""Submit PyHPS jobs to a cluster"""
+
+import logging
+import os
+from typing import Optional, Union
+
+import click
+
+from ansys.mapdl.core.cli import main
+
+logger = logging.getLogger()
+
+
+@main.command(
+    short_help="Submit jobs to an HPC cluster using PyHPS package.",
+    help="""Submit jobs to an HPC cluster using PyHPS package.""",
+)
+@click.argument("main_file")
+@click.option(
+    "--name",
+    default=None,
+    type=str,
+    help="""Name of the PyHPS project to be created.""",
```

Comment:
```suggestion
    help="""Name of the PyHPS project to create.""",
```
Keep it in the present tense.